CHANGELOG.md

```diff
@@ -1,16 +1,18 @@
-# nf-core/staramr: Changelog
+# phac-nml/staramrnf: Changelog
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## [0.1.0] - 2024-12-08
```

Comment:
Can you change to today's date (also format like yyyy-mm-dd).

Reply:
It feels like the year is going by fast but thankfully not that fast! Made the change 4f2d48b